cranelift/codegen/src/data_value.rs

```diff
@@ -249,7 +249,7 @@ impl DataValue {
     ///
     /// Returns true if all bits are equal.
     ///
-    /// This behaviour is different from PartialEq for NaN floats.
+    /// This behavior is different from PartialEq for NaN floats.
```

Comment:
I'd prefer not to have an unrelated change in this PR, and particularly not one that just changes spelling from British English to American English. Let's leave the Commonwealth folks alone. :laughing:

Reply:
Done!